### PR TITLE
Add fade and reply button to view reactions 

### PR DESCRIPTION
**Pull request by alanna-zhou**
### Overview 
- ignoring the dots bc doesn't make sense if we have more than 5 reactions. also it do be kinda hard to do rn

https://user-images.githubusercontent.com/13739525/159196395-7c010546-e21c-474c-9ff8-c3df55d7f8f2.mov

### CODE REVIEW

**Review by haiyingweng — APPROVED** [review]

File: Telie/Controllers/EpisodeReactionsViewController.swift [hunk 1]
```diff
@@ -166,9 +181,8 @@ extension EpisodeReactionsViewController: UICollectionViewDataSource, UICollecti
...
 
 extension EpisodeReactionsViewController: UICollectionViewDelegateFlowLayout {
-
     func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
-      
-        return CGSize(width: collectionView.frame.width, height: collectionView.frame.height)
-    }
+            let width = self.view.frame.width - 40
+            return CGSize(width: width, height: collectionView.frame.height)
+        }
```
[the remark this answers is not in the document]
Comment (haiyingweng) [reply Telie/Controllers/EpisodeReactionsViewController.swift:187]: maye try `.left` or smtg like that

File: Telie.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>BuildSystemType</key>
+	<string>Original</string>
+	<key>PreviewsEnabled</key>
+	<false/>
+</dict>
+</plist>
```
[the remark this answers is not in the document]
Comment (alanna-zhou) [reply Telie.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings:10]: i have no idea LOL should i uncommit this 😂😂😂😂😂

[the remark this answers is not in the document]
Comment (haiyingweng) [reply Telie.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings:10]: I think it ok to leave here

File: Telie/Controllers/EpisodeReactionsViewController.swift (hunk 1 above)
[the remark this answers is not in the document]
Comment (alanna-zhou) [reply Telie/Controllers/EpisodeReactionsViewController.swift:187]: works like a charm

**Pull request merged**